api/wallet/spec/swagger.yaml

```
@@ -1,32 +1,24 @@
 swagger: '2.0'
 info:
-  version: '0.0.1'
-  title: RBKmoney Wallet API
+  version: '0.1.0'
+  title: Valitydev Wallet API
```

```suggestion
  title: Vality Wallet API
```

Здесь и далее похожие места надо б поправить.